ZenPacks/zenoss/ZenPackLib/zenpacklib.py

```diff
@@ -42,8 +42,11 @@
 
 # these are the base classes for zenpacklib.Device, zenpacklib.Component, zenpacklib.HardwareComponent
 from lib.base.Device import Device
-from lib.base.Component import Component
-from lib.base.HardwareComponent import HardwareComponent
+from lib.base.Component import Component, HWComponent, HardwareComponent, CPU, \
```

Comment:
I'd prefer to see the following multi-import style used in alphabetical order in cases like this.

```python
from lib.base.Component import (
    Component,
    HardwareComponent,
    HWComponent,
    etc,
    )
```

It's easier to visually scan for what's imported if they're sorted, and it creates much tidier diffs that result in fewer merge conflicts if they need to be changed.